components/rfid-reader/PC-SC/README.md

```diff
@@ -0,0 +1,22 @@
+# How to setup the PC/SC RFID readers
+
+[PC/SC](https://en.wikipedia.org/wiki/PC/SC) PC/SC (short for "Personal Computer/Smart Card") is a specification for smart-card integration into computing environments.
+
+* https://github.com/MiczFlor/RPi-Jukebox-RFID/pull/1250
+* https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/533
+
+## Install
+
+1. **You can use the `setup_PCSCreader.sh` script (recommended)** or follow the manual steps:
```

Comment:
I thought about removing that manual section in the other components, because it’s probably difficult to keep the Readme and the actual script in sync forever. I know how hard it is to maintain docs up to date